semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Javacp.scala

```diff
@@ -166,7 +166,10 @@ object Javacp {
 
     val classParents = classSignature match {
       case Some(c: ClassSignature) =>
-        c.parents.map(_.toType)
+        try c.parents.map(_.toType)
+        catch {
+          case _: NullPointerException => Nil
```

Comment:
@olafurpg This is pretty outrageous, but I didn't have time to think of a proper solution. Gotta crash now.

Reply:
Every field on asm nodes is nullable, so i'm not surprised one bit. Can take a look.

Reply:
Fixed in #1367